[2023-12-22T14:58:28.925+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_hooks_v02.postgres_to_s3 manual__2023-12-22T14:58:26.460495+00:00 [queued]>
[2023-12-22T14:58:29.004+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_hooks_v02.postgres_to_s3 manual__2023-12-22T14:58:26.460495+00:00 [queued]>
[2023-12-22T14:58:29.020+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2023-12-22T14:58:29.107+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): postgres_to_s3> on 2023-12-22 14:58:26.460495+00:00
[2023-12-22T14:58:29.126+0000] {standard_task_runner.py:60} INFO - Started process 58923 to run task
[2023-12-22T14:58:29.160+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_hooks_v02', 'postgres_to_s3', 'manual__2023-12-22T14:58:26.460495+00:00', '--job-id', '256', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_hooks.py', '--cfg-path', '/tmp/tmpdyo2d5pl']
[2023-12-22T14:58:29.177+0000] {standard_task_runner.py:88} INFO - Job 256: Subtask postgres_to_s3
[2023-12-22T14:58:29.351+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_with_postgres_hooks_v02.postgres_to_s3 manual__2023-12-22T14:58:26.460495+00:00 [running]> on host dd88aaafc1d9
[2023-12-22T14:58:29.523+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sarwar' AIRFLOW_CTX_DAG_ID='dag_with_postgres_hooks_v02' AIRFLOW_CTX_TASK_ID='postgres_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2023-12-22T14:58:26.460495+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-22T14:58:26.460495+00:00'
[2023-12-22T14:58:29.543+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:207 AirflowContextDeprecationWarning: Accessing 'next_ds_nodash' from the template is deprecated and will be removed in a future version. Please use '{{ data_interval_end | ds_nodash }}' instead.
[2023-12-22T14:58:29.561+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-12-22T14:58:29.596+0000] {dag_with_postgres_hooks.py:26} INFO - Saved orders in file: dags/get_orders_20231222.txt
[2023-12-22T14:58:29.600+0000] {python.py:201} INFO - Done. Returned value was: None
[2023-12-22T14:58:29.619+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_hooks_v02, task_id=postgres_to_s3, execution_date=20231222T145826, start_date=20231222T145828, end_date=20231222T145829
[2023-12-22T14:58:29.707+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2023-12-22T14:58:29.742+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
